Utilities/ocvalidate/Checklist.md

```diff
@@ -0,0 +1,65 @@
+- [ ] Patch->Entry[N]->Comment: Only ASCII printable characters are accepted.
+- [ ] Patch->Entry[N]->Arch: Only `Empty string, Any, i386, x86_64` are accepted.
+- [ ] Patch->Entry[N]->Identifier: Only `Empty string, Any, Apple`, or a specified bootloader with `.efi` sufffix, are accepted.
+- [ ] Patch->Entry[N]->Find/Replace/Mask/ReplaceMask: Identical size of these patterns guaranteed.
+- [ ] Quirks->AllowRelocationBlock: When enabled, ProvideCustomSlide should be enabled altogether.
+- [ ] Quirks->AllowRelocationBlock: When enabled, AvoidRuntimeDefrag is highly recommended to be enabled as well.
+- [ ] Quirks->EnableSafeModeSlide: When enabled, ProvideCustomSlide should be enabled altogether.
+- [ ] Quirks->ProvideMaxSlide: If set to a number greater than zero, ProvideCustomSlide should be enabled altogether.
+- [ ] Quirks->DisableVariableWrite/EnableWriteUnprotector: When either is enabled, `OpenRuntime.efi` should be loaded under `UEFI->Drivers`.
+- [ ] Quirks->EnableWriteUnprotector/RebuildAppleMemoryMap: These two cannot be enabled simultaneously.
+- [ ] Quirks->RebuildAppleMemoryMap: When enabled, SyncRuntimePermissions is highly recommended to be enabled as well.
+
+## DeviceProperties
+- [ ] TODO
+
+## Kernel
+- [ ] Add->Entry[N]->Arch: Only `Any, i386, x86_64` are accepted.
+- [ ] Add->Entry[N]->BundlePath: Only `0-9, A-Z, a-z, '_', '-', '.', '/', and '\'` are accepted.
+- [ ] Add->Entry[N]->Comment: Only ASCII printable characters are accepted.
+- [ ] Add->Entry[N]->ExecutablePath: Only `0-9, A-Z, a-z, '_', '-', '.', '/', and '\'` are accepted.
+- [ ] Add->Entry[N]->PlistPath: Only `0-9, A-Z, a-z, '_', '-', '.', '/', and '\'` are accepted.
+- [ ] Delete->Entry[N]->Arch: Only `Any, i386, x86_64` are accepted.
+- [ ] Delete->Entry[N]->Comment: Only ASCII printable characters are accepted.
+- [ ] Delete->Entry[N]->Identifier: Only `0-9, A-Z, a-z, '_', '-', '.', '/', and '\'` are accepted.
+
+## Misc
+- [ ] TODO
+
+## NVRAM
+- [ ] TODO
+
+## PlatformInfo
+- [ ] TODO
+
+## UEFI
+- [ ] APFS->EnableJumpstart: When enabled, Misc->Security->ScanPolicy should have OC_SCAN_ALLOW_FS_APFS (bit 8) set.
+- [ ] Quirks->RequestBootVarRouting: When enabled, `OpenRuntime.efi` should be loaded under `UEFI->Drivers`.
+- [ ] Quirks->DeduplicateBootOrder: When enabled, RequestBootVarRouting should be enabled altogether.
+- [ ] Driver OpenUsbKbDxe.efi: When in use, UEFI->Input->KeySupport should never be enabled altogether.
+- [ ] Driver Ps2KeyboardDxe.efi: When in use, UEFI->Input->KeySupport should be enabled altogether.
+- [ ] OpenUsbKbDxe.efi and Ps2KeyboardDxe.efi should never co-exist.
+- [ ] Output->ClearScreenOnModeSwitch/IgnoreTextInGraphics/ReplaceTabWithSpace/SanitiseClearScreen: These only apply to `System` TextRenderer
+- [ ] Output->SanitiseClearScreen: When enabled, `ConsoleMode` should be empty.
+- [ ] Output->ConsoleMode: Only ASCII printable characters are accepted.
+- [ ] Output->TextRenderer: Only ASCII printable characters are accepted.
```

Comment:
Add If `PointerSupport` is enabled, then `PointerSupportMode` should be set to `ASUS`.

Reply:
KeySupportMode should be one of `Auto`, `V1`, `V2`, `AMI`.

Reply:
Done. https://github.com/PMheart/OpenCorePkg/commit/11d8a16b2041aa005015e8ef9ca088da43bbd279 and https://github.com/PMheart/OpenCorePkg/commit/f25ab365898d68be6d8b69d5d53255637e2b94b6